<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>amphetype2</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="Quizzer" name="quizzer">
       <attribute name="title">
        <string>Typer</string>
       </attribute>
      </widget>
      <widget class="TextManager" name="textManager">
       <attribute name="title">
        <string>Sources</string>
       </attribute>
      </widget>
      <widget class="PerformanceHistory" name="performanceHistory">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
      </widget>
      <widget class="StatisticsWidget" name="statisticsWidget">
       <attribute name="title">
        <string>Analysis</string>
       </attribute>
      </widget>
      <widget class="TrainingGenWidget" name="trainingGenWidget">
       <attribute name="title">
        <string>Training Generator</string>
       </attribute>
      </widget>
      <widget class="SettingsWidget" name="settingsWidget">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPlot"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="plotDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>plot</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="LivePlot" name="plot"/>
  </widget>
  <action name="actionPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Quizzer</class>
   <extends>QWidget</extends>
   <header>quizzer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextManager</class>
   <extends>QWidget</extends>
   <header>textmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerformanceHistory</class>
   <extends>QWidget</extends>
   <header>performancehistory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatisticsWidget</class>
   <extends>QWidget</extends>
   <header>statisticswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>settingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingGenWidget</class>
   <extends>QWidget</extends>
   <header>traininggenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LivePlot</class>
   <extends>QWidget</extends>
   <header>liveplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
